lantern_django/README.md

```diff
@@ -86,3 +86,14 @@ class Item(models.Model):
             )
         ]
 ```
+
+
+### Running the test_project and it's tests
```

Comment:
Remove from README, since this isn't developer facing (it's for people maintaining lantern-django)